met-api/src/met_api/models/engagement.py

```diff
@@ -117,6 +118,9 @@ def update_engagement(cls, engagement: EngagementSchema) -> DefaultMethodResult:
     @classmethod
     def close_engagements_due(cls) -> List[EngagementSchema]:
         """Update engagement to closed."""
+        now = local_datetime()
+        date_due = (now + timedelta(days=-1))
```

Comment:
I know it's overhead, but I don't feel confident I understand what's happening here. Maybe a brief clarifying comment?